inc/redis_alloptions/namespace.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace HM\Platform\Redis_Alloptions;
```

Comment:
I'm thinking we don't need to actually call this namespace "redis", just only load it when using redis. maybe `Alloptions_Fix` or something?